tests/network/vmi_networking.go

```diff
@@ -719,8 +719,8 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				Expect(libnet.PingFromVMConsole(serverVMI, ipAddr)).To(Succeed())
 
 				By("Checking ping (IPv4) to google")
-				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8")).To(Succeed())
-				Expect(libnet.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
+				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8", "-c 5", "-w 20")).To(Succeed())
```

Comment:
Do you know if the timeout hit the overall limit or limit per ping?

Reply:
w here is deadline, its for the whole packets,

Its also answer about https://github.com/kubevirt/kubevirt/pull/5430#discussion_r611568246 comment,
no need to c*w, just w, and we can even just extend it to 20, and have "-w 15" in the specific tests, which would be enough, and consume the min of (w, max 20) just if the test fails, wdyt?
In other PR we can improve the mechanism if needed, as its not related to the flake this PR tries to fix